services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/PathHelper.groovy

```diff
@@ -87,8 +89,8 @@ class PathHelper {
     /**
      * Converts FlowPathPayload path representation to a List<PathNode> representation
      */
-    static List<PathNode> convert(FlowPathPayload pathPayload) {
-        def path = pathPayload.forwardPath
+    static List<PathNode> convert(FlowPathPayload pathPayload, pathToConvert = "forwardPath") {
```

Comment:
If I understand correctly, this method previously converted the both flows. Are you sure it won't affect existing tests? Maybe it'd be better to create a separate method?

Reply:
This method previously converted the forward path only. Please take a look at L93. Now it can convert both paths but the forward path is converted by default. So it shouldn't break existing tests.